cfgov/cfgov/settings/base.py

```diff
@@ -274,3 +328,26 @@
     AWS_S3_CALLING_FORMAT = 'boto.s3.connection.OrdinaryCallingFormat'
 
     MEDIA_URL = os.environ.get('AWS_S3_URL') + '/f/'
+
+# Govdelivery
+
+GOVDELIVERY_USER = os.environ.get('GOVDELIVERY_USER')
+GOVDELIVERY_PASSWORD = os.environ.get('GOVDELIVERY_PASSWORD')
+GOVDELIVERY_ACCOUNT_CODE = os.environ.get('GOVDELIVERY_ACCOUNT_CODE')
+
+# LOAD OPTIONAL APPS
+# code from https://gist.github.com/msabramo/945406
+
+for app in OPTIONAL_APPS:
+    if app.get("condition", True):
+	try:
+	    __import__(app["import"])
+	except ImportError:
+	    pass
+	else:
+	    for name in app.get("apps",()):
+		if name not in INSTALLED_APPS:
```

Comment:
@rosskarchner ok now i understand. the main issue was i hadn't seen an `else` used after a `try` before had to look it up https://docs.python.org/2/tutorial/errors.html to understand what was going on.